src/nvim/indent_c.c

```diff
@@ -513,23 +513,28 @@ static int cin_isdefault(const char_u *s)
 }
 
 /*
- * Recognize a "public/private/protected" scope declaration label.
+ * Recognize a scope declaration label set in 'cinscopedecls'.
  */
 bool cin_isscopedecl(const char_u *s)
```

Comment:
Rename to `p`

Reply:
Sorry, rename what exactly?

Reply:
The argument is renamed from `s` to `p` in the patch.